packages/react-searchbox/src/components/SearchComponent.js

```diff
@@ -199,16 +230,21 @@ class SearchComponent extends React.Component {
         this.setState(state);
       }, subscribeTo);
     }
-    if (value || customQuery) {
-      this.componentInstance.triggerCustomQuery();
-    }
   }
 
   componentDidMount() {
-    const { triggerQueryOnInit } = this.props;
-
-    if (triggerQueryOnInit) {
-      this.componentInstance.triggerDefaultQuery();
+    const { triggerQueryOnInit, customQuery } = this.props;
+    if (this.componentInstance) {
+      if (triggerQueryOnInit) {
+        this.componentInstance.triggerDefaultQuery();
+      }
+      if (
+        // check to prevent customQuery at clientside when leveraging SSR
+        (!this.context.initialState && this.componentInstance.value) ||
```

Comment:
Update the condition.
(this.context.initialState && (this.componentInstance.value || customQuery))